types/src/schemas/storage_service/user_nft_list_with_offset.rs

```diff
@@ -0,0 +1,16 @@
+use runtime::AccountId;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct GetUserNftListWithOffsetSchema {
+    #[serde(rename = "accountId")]
+    account_id: AccountId,
+    id: String,
+    params: Params,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Params {
```

Comment:
Rename to `GetUserNftListWithOffsetSchemaParams`. Params struct name = RequestName + Params